optimizely/config_manager.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2019, Optimizely
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import http
+import requests
+import threading
+import time
+from requests import exceptions as requests_exceptions
+
+from optimizely import exceptions
+from optimizely import logger as _logging
```

Comment:
nit: can this be `optimizely_logger` instead of `logging`?

Reply:
Yeah, will change that.